Library/Formula/globus-toolkit.rb

```diff
@@ -12,14 +10,35 @@ class GlobusToolkit < Formula
     sha1 "1cbbe7a3d08711cbc5665e91558417d7b2028d3a" => :mountain_lion
   end
 
+  depends_on "openssl"
   depends_on "libtool" => :run
+  depends_on "pkg-config" => :build
+
+  option "with-check", "Test the toolkit when installing"
 
   def install
     ENV.deparallelize
     ENV["MACOSX_DEPLOYMENT_TARGET"] = MacOS.version
-    system "./configure", "--prefix=#{prefix}"
+    man.mkpath
+    system "./configure", "--prefix=#{prefix}",
```

Comment:
If you just set this to ` #{libexec} ` you can avoid renaming the header and won't have to deal with so much of the manually moving stuff around below. The bin symlink is usually enough for things to find the dep, and if not, we can insert flags in other formulae.